shell/common/pipeline.h

```diff
@@ -111,16 +111,20 @@ class Pipeline : public fml::RefCountedThreadSafe<Pipeline<R>> {
         GetNextPipelineTraceID()};         // trace id
   }
 
-  // Pushes task to the front of the pipeline.
-  //
-  // If we exceed the depth completing this continuation, we drop the
-  // last frame to preserve the depth of the pipeline.
-  //
-  // Note: Use |Pipeline::Produce| where possible. This should only be
-  // used to en-queue high-priority resources.
-  ProducerContinuation ProduceToFront() {
+  // Create a `ProducerContinuation` that will only push the task if the queue
+  // is empty.
```

Comment:
It would be useful to add a brief description for the recommended usage of this method. Something along the lines of:

```
Prefer using Produce. ProducerContinuation returned by this method doesn't  guarantee that the frame will be rendered.
```

Reply:
done